core/src/worker/workflow/mod.rs

```diff
@@ -829,6 +829,7 @@ struct RequestEvictMsg {
     run_id: String,
     message: String,
     reason: EvictionReason,
+    because_fetch_failed: bool,
```

Comment:
Could that special purpose field have been avoided? I don't know if adding a new `EvictionReason` would be better?

Reply:
Good point, I'll change that